streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsProducer.java

```diff
@@ -223,7 +224,9 @@ private double totalBlockedTime(final Producer<?, ?> producer) {
             + getMetricValue(producer.metrics(), "txn-begin-time-ns-total")
             + getMetricValue(producer.metrics(), "txn-send-offsets-time-ns-total")
             + getMetricValue(producer.metrics(), "txn-commit-time-ns-total")
-            + getMetricValue(producer.metrics(), "txn-abort-time-ns-total");
+            + getMetricValue(producer.metrics(), "txn-abort-time-ns-total")
```

Comment:
This is a meta question: since ns and ms measurement mechanisms differ, if we simply sum them as `ns + ms * 10^6` it may not be accurate. Probably worth checking in if the aggregated total blocked time does make sense.